CSS/Components/navigation/navigation.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>navigation</title>
+    <link rel="stylesheet" href="navigation.css">
+</head>
+<body>
+
+    <nav class="navbar bg-red">
+    
+        <div class="navbar-header">
+
+            <a href="#"><p class="navbar-header-title text-l text-white">Navbar Title</p></a>
+            
+        </div>
+
+        <div class="navbar-links">
+
+            <a href="#"><img src="../../../assets/download_white_36dp.svg" alt="download"></a>
+            <a href="#"><img src="../../../assets/print_white_36dp.svg" alt="print"></a>
+            <a href="#"><img src="../../../assets/bookmark_white_36dp.svg" alt="bookmark"></a>
+
+        </div>
+        
+    </nav>
+    
```

Comment:
You can remove the extra spaces and format the file

Reply:
done